korman/ui/ui_world.py

```diff
@@ -77,10 +121,26 @@ def draw(self, context):
         col = split.column()
         col.label("Age Settings:")
         col.prop(age, "seq_prefix", text="ID")
+        col.prop(age, "age_name", text="")
```

Comment:
We should set the `alert` property on this one if the value is currently empty, so that it calls attention to the invalid name.